app/Http/Controllers/MgtController.php

```diff
@@ -561,6 +584,21 @@ public function addLog(Request $request) {
         $le->log = $request->log;
         $le->save();
 
+        EmailHelper::sendEmail(
+            [
+                Auth::user()->email,
```

Comment:
Probably don't want to send to the submitting user. They already know what they just wrote, and it'll just be another email that people try to filter out as spam.

Reply:
Double checked with Barrett and he wants it to be sent back to the member so they have a receipt to forward in the future if needed.